app.js

```diff
@@ -0,0 +1,77 @@
+// constructor for object
+class Book {
+  KEY='data';
+
+  constructor() {
+    this.collection = this.collectionMethod();
+    this.ul = document.querySelector('.lst');
+    this.title = document.querySelector('.title-input');
+    this.author = document.querySelector('.author-input');
+    this.addBtn = document.querySelector('.add-btn');
+  }
+
+  collectionMethod() {
+    const data = localStorage.getItem(this.KEY);
+    return data ? JSON.parse(data) : [];
+  }
+
+  saveLocal(arr) {
+    localStorage.setItem(this.KEY, JSON.stringify(arr));
+  }
+
+  getfromLocal() {
+    const arr = JSON.parse(localStorage.getItem(this.KEY));
+    return arr;
+  }
+
+  clearPlaceHolder() {
+    this.title.value = '';
+    this.author.value = '';
+  }
+
+  removeBtn(e) {
+    const li = e.target.parentElement;
+    this.collection = this.collection.filter((book) => book.title !== li.id);
+    this.saveLocal(this.collection);
+    li.remove();
+  }
+
+  createList(element) {
+    // const newBook = this.getfromLocal();
```

Comment:
- [ ] Good job creating the book successfully :clap: , However, kindly remove this commented-out line of code to make your code cleaner :recycle: